packages/teleterm/package.json

```diff
@@ -17,7 +17,7 @@
     "build-main": "webpack build --config webpack.main.config.js --progress --mode=production",
     "build-renderer": "webpack build --config webpack.renderer.prod.config.js --progress",
     "build-natives": "electron-builder install-app-deps",
-    "package": "electron-builder build --publish never -c.extraMetadata.name=teleterm"
+    "package": "electron-builder build --publish never -c.extraMetadata.name=teleconnect"
```

Comment:
[There was an issue with `productName` on Linux](https://github.com/electron-userland/electron-builder/issues/1567). For now I will change it to `teleconnect`, but we will verify that when we will release the app for that OS.